src/Message/ButtonReply/ButtonAction.php

```diff
@@ -27,4 +34,25 @@ public function buttons(): array
 
         return $buttonActions;
     }
+
+    /**
+     * Get the header of the button action, if it exists
+     *
+     * @return array|null The header of the button action, or null if it doesn't exist
```

Comment:
the header function don't allow return nullable values as you have declared it.

Reply:
Same with above.